Lib/tracemalloc.py

```diff
@@ -221,7 +226,12 @@ def __str__(self):
         return str(self[0])
 
     def __repr__(self):
-        return "<Traceback %r>" % (tuple(self),)
+        s = "<Traceback %r" % tuple(self)
+        if self._total_nframe is None:
+            s += ">"
+        else:
+            s += "total_nframe=%d>" % self.total_nframe
```

Comment:
You forgot a space. I like f-string :-) No need to be pedantic with %d.

```suggestion
            s += f" total_nframe={self.total_nframe}>"
```

Reply:
Thanks!